### PR TITLE
Update auth client to handle response from superagent

### PR DESCRIPTION
**Pull request by edpaget**
Depends on zooniverse/json-api-client#14

### CODE REVIEW

**Comment by edpaget** [comment]
This this should all set if we want to do it.

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "react-select": "marten/react-select#close-after-click",
     "react-translate-component": "~0.9.0",
     "sugar-client": "^1.0.1",
+    "superagent": "^1.3.0",
```
Comment (chrissnyder) [inline package.json]: Don't think this is necessary.

**Comment by chrissnyder** [comment]
Looks good to me after a rebase.

**Comment by edpaget** [comment]
Done

**Pull request merged**